ansible/roles/cartodb.app/templates/cartodb_server.conf.j2

```diff
@@ -0,0 +1,17 @@
+# WARNING: This file is auto-generated by Ansible. Any changes will
+# be overwritten the next time ansible is run.
+
+description "Start the CartoDB rails development server"
+
+stop on runlevel [!2345]
+
+respawn
+# Kill respawn if more than 5 restarts in 20 mins
+respawn limit 5 20
+
+script
+    APP_ROOT={{ cartodb_dir }}
+
+    cd $APP_ROOT
```

Comment:
Might be better to use `chdir` here vs. depending on the ability to `cd` inside a `script` block.